include/uapi/sound/sof/tokens.h

```diff
@@ -57,6 +57,12 @@
 #define SOF_TKN_SRC_RATE_IN			300
 #define SOF_TKN_SRC_RATE_OUT			301
 
+/* ASRC */
+#define SOF_TKN_ASRC_RATE_IN			302
+#define SOF_TKN_ASRC_RATE_OUT			303
```

Comment:
@ranj063 @plbossart Here I created duplicated token codes for ASRC. I think nothing prevents to reuse values 300 and 301 if we prefer and maybe reuse even the same macros. What do you think is best?

Reply:
@singalsu im ok both ways. might be better to re-use the old ones

Reply:
Decided to have new ones. Also I per FW review feedback left a gap in numbering to add more tokens for the old SRC.